DataConnectors/JumpCloud Single Sign On/azuredeploy_JumpCloud_API_FunctionApp.json

```diff
@@ -224,7 +224,7 @@
                         },
                         {
                             "name": "WEBSITE_RUN_FROM_PACKAGE",
-                            "value": "https://github.com/cabberley/JumpCloudSSO/raw/master/JumpCloudSSO.zip"
+                            "value": "https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/JumpCloud%20Single%20Sign%20On/AzureFunctionJumpCloud/JumpCloudSSO.zip?raw=true"
```

Comment:
@v-sabiraj : Please create a short link and use it here

Reply:
@v-rucdu, updated, thanks.